From b3e0fdadf5a6b0a7facfb10c60166ebe8039d753 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Thu, 12 Dec 2024 00:29:07 +0800
Subject: [PATCH 278/300] AOSCOS: MIPS: math-emu: replace CPU_LOONGSON3
 conditions with CPU_LOONGSON64

Per commit 30ad29bb4888 ("MIPS: Loongson: Naming style cleanup and
rework"), rename the CPU_LOONGSON3 definition as CPU_LOONGSON64.

Fixes: "FROMLIST: MIPS: math-emu: Add madd/msub/nmadd/nmsub emulation for Loongson-3"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/math-emu/cp1emu.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/mips/math-emu/cp1emu.c b/arch/mips/math-emu/cp1emu.c
index a2cba79eba7a..0c618683e57c 100644
--- a/arch/mips/math-emu/cp1emu.c
+++ b/arch/mips/math-emu/cp1emu.c
@@ -1451,7 +1451,7 @@ static union ieee754sp fpemu_sp_rsqrt(union ieee754sp s)
 	return ieee754sp_div(ieee754sp_one(0), ieee754sp_sqrt(s));
 }
 
-#ifndef CONFIG_CPU_LOONGSON3
+#ifndef CONFIG_CPU_LOONGSON64
 DEF3OP(madd, sp, ieee754sp_mul, ieee754sp_add, );
 DEF3OP(msub, sp, ieee754sp_mul, ieee754sp_sub, );
 DEF3OP(nmadd, sp, ieee754sp_mul, ieee754sp_add, ieee754sp_neg);
@@ -1515,7 +1515,7 @@ static int fpux_emu(struct pt_regs *xcp, struct mips_fpu_struct *ctx,
 			}
 			break;
 
-#ifdef CONFIG_CPU_LOONGSON3
+#ifdef CONFIG_CPU_LOONGSON64
 		case madd_s_op:
 			handler = ieee754sp_madd;
 			goto scoptop;
@@ -1638,7 +1638,7 @@ static int fpux_emu(struct pt_regs *xcp, struct mips_fpu_struct *ctx,
 			}
 			break;
 
-#ifdef CONFIG_CPU_LOONGSON3
+#ifdef CONFIG_CPU_LOONGSON64
 		case madd_d_op:
 			handler = ieee754dp_madd;
 			goto dcoptop;
-- 
2.48.1

